### ISSUE TITLE
[Question][1.16.5] Heat Sources

### ISSUE THREAD

**Issue by drakray** [elucent/eidolon#114]
I see in the code these as Heat Source:
MAGMA_BLOCK
FIRE
SOUL_FIRE
LAVA
CAMPFIRE.LIT
SOUL_CAMPFIRE.LIT

Is there a way to configure more source?
Either by Config or blockTags?
I'd like some modded source to count, like SoulFire from Druidcraft, SpiritFire from Occultism, Campfires from OhTheBiomesYoullGo and EndergeticExpansion, and FirePlace from Vampirism

Something like blockTag:forge:heat_source would make that easily configurable with CraftTweaker, for example, and make modder able to implement it by default if they want

**Comment by Alexthw46** [comment]
About campfires, the solution can be easily achived replacing the vanilla check with a generic (instance of CampfireBlock && [isLit] ).

I added that line in Hexblades update, but having a tag here in Eidolon would be easier and more compatible.

**Comment by drakray** [comment]
Yeah, I think Elucent should make the Crucible use blockTags, something like forge:heat_source, it would make it MUCH easier to adapt in mods and modPack using CraftTweaker
